keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -277,65 +269,73 @@ the `/resize` subresource:
-* RestartContainer - the container requires a restart to apply new resource values.
+* `PreferNoRestart` - default value; resize the Container without restart, if possible.
+  * `NotRequired` - Equivalent to `PreferNoRestart`, deprecated with v1.33.
+* `RestartContainer` - the container requires a restart to apply new resource values.
   (e.g.  Java process needs to change its Xmx flag) By using ResizePolicy, user
   can mark Containers as safe (or unsafe) for in-place resource update. Kubelet
   uses it to determine the required action.
 
-Note: `NotRequired` restart policy for resize does not *guarantee* that a container
-won't be restarted. The runtime may choose to stop the container if it is unable to
-apply the new resources without restarts.
+Note: `PreferNoRestart` restart policy for resize does not *guarantee* that a container
+won't be restarted.
 
 Setting the flag to separately control CPU & memory is due to an observation
 that usually CPU can be added/removed without much problem whereas changes to
 available memory are more probable to require restarts.
 
 If more than one resource type with different policies are updated at the same
-time, then `RestartContainer` policy takes precedence over `NotRequired` policy.
+time, then `RestartContainer` policy takes precedence over `PreferNoRestart` policy.
 
 If a pod's RestartPolicy is `Never`, the ResizePolicy fields must be set to
-`NotRequired` to pass validation.  That said, any in-place resize may result
+`PreferNoRestart` to pass validation.  That said, any in-place resize may result
 in the container being stopped *and not restarted*, if the system can not
 perform the resize in place.
 
-The `ResizePolicy` field is mutable. Only append is strictly required to support in-place resize
-(for new resource requirements), and we may reevaluate full mutability if additional edge cases are
-discovered in implementation.
+The `ResizePolicy` field is immutable.
 
 #### Resize Status
 
-In addition to the above, a new field `Pod.Status.Resize[]`
-will be added.  This field indicates whether kubelet has accepted or rejected a
-proposed resize operation for a given resource.  Any time the
-`Pod.Spec.Containers[i].Resources.Requests` field differs from the
-`Pod.Status.ContainerStatuses[i].Resources` field, this new field explains why.
-
-This field can be set to one of the following values:
-* `Proposed` - the proposed resize (in Spec...Resources) has not been accepted or
-  rejected yet. Desired resources != Allocated resources.
-* `InProgress` - the proposed resize has been accepted and is being actuated. A new resize request
-  will reset the status to `Proposed`. Desired resources == Allocated resources != Actual resources.
+Resize status will be tracked via 2 new pod conditions: `PodResizePending` and `PodResizing`.
+
+**PodResizePending** will track states where the spec has been resized, but the Kubelet has not yet
```

Comment:
When the spec has been resized, but Kubelet hasn't evaluated the resizing request yet, what's the pod condition then?

Reply:
Previously newly introduced field `Proposed` indicates this state, I am not sure the alternative in the new proposal.

Reply:
nm. You explained this below.